bundle/manifests/observability-operator.clusterserviceversion.yaml

```diff
@@ -852,15 +852,11 @@ spec:
   - prometheus
   - thanos
   links:
-  - name: Observability Operator
-    url: https://rhobs-handbook.netlify.app/products/observability-operator
   - name: GitHub
     url: https://github.com/rhobs/observability-operator
   maintainers:
   - email: jfajersk@redhat.com
     name: Jan Fajerski
-  - email: sthaha@redhat.com
-    name: Sunil Thaha
```

Comment:
In the light of this change, should https://github.com/rhobs/observability-operator/blob/a52ed9be7db4c346adbd54cfe4aa59d6324584fe/hack/olm/catalog-src.yaml#L15 point to the team instead?

Reply:
It doesn't apply for the productized version because the operator is shipped via the Red Hat catalog but I can fix in a following PR.